src/protocol_message.rs

```diff
@@ -1,17 +1,30 @@
 //! Protocol messages exchanged between swap daemons
 
-use crate::crypto::Proof;
+use crate::crypto::{Commitment, Proof, Signatures, Curve};
 use crate::role::{Accordant, Arbitrating};
 
 /// Trait for defining inter-daemon communication messages.
 pub trait ProtocolMessage {}
 
+use strict_encoding::{StrictDecode, StrictEncode};
+
+pub trait ProtMsg {}
```

Comment:
Why `ProtMsg` if we already have `ProtocolMessage`?

Reply:
fixed in 0b95fcc